### PR TITLE
aggregate_test verify DUT oper-status before configure ATE

### PR DESCRIPTION
**Pull request by liulk**

### CODE REVIEW

**Comment by liulk** [comment]
Invocation ID: 659613f9-9e31-4d4d-be08-7a0107e65780

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 1]
```diff
@@ -294,40 +292,47 @@ const (
...
 	}
 }
+
 func (tc *testCase) verifyInterfaceDUT(t *testing.T, dp *ondatra.Port) {
 	dip := tc.dut.Telemetry().Interface(dp.Name())
 	di := dip.Get(t)
 	fptest.LogYgot(t, dp.String(), dip, di)
+
+	// LAG members may fall behind, so wait for them to be up.
+	dip.OperStatus().Await(t, time.Minute, opUp)
```
Comment (bstoll) [inline feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go:309]: di needs to be collected after the Await (or just use the values returned by Await) doesn't it?  If the Await is doing anything, the values saved in di are the old states, and they will just fail in the next few lines even if the interface does come up after a delay.

Comment (liulk) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go:309]: But if I move the dip.Get(t) after the await, then if the interface is down, then await will fatal out and di is never logged.

Comment (liulk) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go]: Removed the di.GetOperStatus() check which used potentially stale value.

**Review by bstoll — APPROVED** [review]

**Pull request merged**